README.md

```diff
@@ -33,11 +33,18 @@ If you are using `dotnet` or `nuget`, you can use the Azure Artifact Credential
 ### Manual installation: Windows
 
 1. Download the latest release of [Microsoft.NuGet.CredentialProvider.zip](https://github.com/Microsoft/artifacts-credprovider/releases)
-1. Unzip the file
-1. Copy the `netcore` directory from the extracted archive to `$env:UserProfile\.nuget\plugins`
+2. Unzip the file
+3. Copy the `netcore` directory from the extracted archive to `$env:UserProfile\.nuget\plugins` to use `dotnet`
+4. Copy the `netfx` directory from the extracted archive to `$env:UserProfile\.nuget\plugins` to use `nuget`
 
 Note: copying the `netfx` directory is not recommended unless you don't plan to install an edition of Visual Studio (including the Build Tools edition). All Visual Studio editions come with (and automatically update) the `netfx` version of the Credential Provider.
 
+### Automatic PowerShell installation:
+
+[PowerShell helper script](helpers/installcredprovider.ps1)
+To install both netfx and netcore, run `installcredprovider.ps1`
```

Comment:
Per the note on the script, can we make netcore only the default, and netfx installable by `-AddNetfx true`?

Reply:
done